frontend/src/App.tsx

```diff
@@ -141,34 +145,40 @@ function App() {
         <div className={styles.Content}>
           <Router>
             <ScrollToTop />
-            <AppBar position="static" elevation={0}>
-              {width > MOBILE_BREAKPOINT ? (
-                <AppToolbar />
-              ) : (
-                <MobileAppToolbar />
-              )}
-            </AppBar>
-            <Switch>
-              <Route path={ABOUT_US_PAGE_LINK} component={AboutUsPage} />
-              <Route path={DATA_CATALOG_PAGE_LINK} component={DataCatalogTab} />
-              <Route
-                path={EXPLORE_DATA_PAGE_LINK}
-                component={ExploreDataPage}
-              />
-              <Route
-                path={WHAT_IS_HEALTH_EQUITY_PAGE_LINK}
-                component={WhatIsHealthEquityPage}
-              />
-              <Route
-                  path={TERMS_OF_SERVICE_PAGE_LINK}
-                  component={TermsOfServicePage}
-              />
-              <Route exact path="/" component={LandingPage} />
-              <Route component={NotFoundPage} />
-            </Switch>
+            <header aria-label="Header content including site navigation">
+              <AppBar position="static" elevation={0}>
+                {width > MOBILE_BREAKPOINT ? (
+                    <AppToolbar />
+                ) : (
+                    <MobileAppToolbar />
+                )}
+              </AppBar>
+            </header>
+            <main>
+              <Switch>
+                <Route path={ABOUT_US_PAGE_LINK} component={AboutUsPage} />
+                <Route path={DATA_CATALOG_PAGE_LINK} component={DataCatalogTab} />
+                <Route
+                    path={EXPLORE_DATA_PAGE_LINK}
+                    component={ExploreDataPage}
+                />
+                <Route
+                    path={WHAT_IS_HEALTH_EQUITY_PAGE_LINK}
+                    component={WhatIsHealthEquityPage}
+                />
+                <Route
+                    path={TERMS_OF_SERVICE_PAGE_LINK}
+                    component={TermsOfServicePage}
+                />
+                <Route exact path="/" component={LandingPage} />
+                <Route component={NotFoundPage} />
+              </Switch>
+            </main>
           </Router>
         </div>
-        <Footer />
+        <footer aria-label="Footer content">
```

Comment:
It is my understanding that you would not want an aria-label here. I think just using the <footer> element type is enough to provide the page structure to assistive technologies.